src/App.tsx

```diff
@@ -11,6 +12,7 @@ function App() {
       <BackendConnectionChecker />
       <Routes>
         <Route path="/" element={<></>} />
+        <Route path="/SpeciesIdentification" element={<SpeciesIdentification />} />
```

Comment:
Best practice would make the path `"/species-identification"` using kebab case. I will raise this path naming at stand-up as I don't think this was discussed during planning

Reply:
amended